.github/workflows/Linting.yml

```diff
@@ -1,38 +1,28 @@
-name: Linting
-
+name: Run Linting on Pull Request and Branches
 on:
   pull_request:
     types:
       - opened
       - synchronize
       - reopened
-
-concurrency:
-  group: ${{ github.workflow }}-${{ github.ref }}
-  cancel-in-progress: true
-
-#Run npm run linting
 jobs:
-  lint:
-    name: 🧹 Lint
+  eslint:
     runs-on: ubuntu-latest
 
     steps:
-      - name: Checkout
+      - name: Check out code
         uses: actions/checkout@v2
 
-      - name: Use Node.js 16.x
-        uses: actions/setup-node@v1
+      - name: Setup Node.js
+        uses: actions/setup-node@v2
         with:
-          node-version: 16.x
+          node-version: 14
+      
+      - name: List directories and files
```

Comment:
Why are we running this command?

Reply:
If you want to you can easily run it in your terminal. My question is to why we would run this in the GitHub Action

Reply:
@Jatewo i removed it before merging :)))